docs/src/components/main.tsx

```diff
@@ -0,0 +1,27 @@
+import { HTMLAttributes, ReactNode } from 'react';
+import { H1_ID } from '../constants';
+
+type MainProps = {
+  children: ReactNode;
+  className?: string;
+  isFocusable?: boolean;
+} & Omit<
+  HTMLAttributes<HTMLElement>,
+  'children' | 'className' | 'tabIndex' | 'aria-labelledby'
+>;
+
+export const Main = ({
+  children,
+  className,
+  isFocusable = true,
+  ...props
+}: MainProps) => (
+  <main
+    className={`outline-0 ${className || ''}`}
```

Comment:
I'm still getting an outline on ios:

![image](https://github.com/Thinkmill/keystatic/assets/5933897/0aaa0ef7-8a3a-4ca0-b024-745ff6c179a0)

Looks like it is a safari issue 😩
![CleanShot 2023-07-07 at 10 38 42@2x](https://github.com/Thinkmill/keystatic/assets/5933897/96585d84-5027-4163-a1b1-dc3dcfaa5093)

Reply:
Ahh bummer, thanks for spotting this! I'll take a look.

Reply:
Switching from `outline-0` to `outline-none` seems to have fixed it.